File: Python/factorial.py
#program to find the factorial of a number
import os
num       =eval(input("enter the number which you want to find the factorial of \n"))
factorial =1
for i in range(num,1,-1) :
	factorial *=i
print("factorial of the number ", num, "is", factorial)

os.system("pause")

File: Cpp/Lessons/factorial.cpp
#include<iostream>
using namespace std;
int main(){
	int i,x;
	cout<<"enter the number\n";
	cin>>i;
	x=i;
	do{
		x=x*(i-1);
		i--;
	}while(i>1);
	cout<<"The factorial is "<<x<<endl;
	return 0;
}
